quinn-proto/src/crypto/rustls.rs

```diff
@@ -256,6 +261,8 @@ pub struct HandshakeData {
     ///
     /// Always `None` for outgoing connections
     pub server_name: Option<String>,
+    /// The key exchange group negotiated with the peer
+    pub negotiated_key_exchange_group: NamedGroup,
```

Comment:
Sorry, we don't want to land this on main right now due to semver compatibility issues.

Reply:
Which option do you prefer: 
- Remove the commit that adds this. In that case I will only test that the handshake succeeds, not the specific key exchange group.
- Keep this PR open until the next breaking release

Reply:
First option sounds okay to me, I'll shortly reopen a PR to add back the API and then we can tack on a commit to add this back to the test as well?

Reply:
@stormshield-damiend suggested, as we use a `__rustls-post-quantum-test` for testing, maybe I can declare this field under this feature flag?

Reply:
Ohh, that makes sense to me, yes!

Reply:
I did this in my last push :tada: